var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -89,6 +89,8 @@ class PyAwkward(PythonPackage):
     depends_on("rapidjson")
     depends_on("cmake@3.13:", type="build")
     depends_on("py-wheel@0.36.0:", when="@:1.7.0", type="build")
+    depends_on("py-importlib-metadata@4.13.0:", when="@2.4.0:", type=("build", "run"))
+    depends_on("py-fsspec@2022.11.0:", when="@2.6.1:", type=("build", "run"))
```

Comment:
Unversioned `py-fsspec` was [added](https://github.com/scikit-hep/awkward/commit/b749e49d6fe078e530ef506bf526705bab3f35f1) as requirement in 2.6.0. I'd add it already for that version.
```suggestion
    depends_on("py-fsspec@2022.11.0:", when="@2.6.0:", type=("build", "run"))
```

Reply:
Done